vernac/comDefinition.ml

```diff
@@ -110,6 +110,10 @@ let interp_definition ~program_mode env evd impl_env bl red_option c ctypopt =
   let tyopt = Option.map (fun ty -> EConstr.it_mkProd_or_LetIn ty ctx) tyopt in
   evd, (c, tyopt), imps
 
+let definition_using env evd ~body ~types ~using =
+  let terms = body :: match types with Some x -> [x] | None -> [] in
```

Comment:
The order of terms is irrelevant so this can be written
```suggestion
  let terms = Option.List.cons types [body]
```
or
```suggestion
  let terms = body :: Option.List.cons types []
```

Reply:
Done, thanks!